chain/jsonrpc/openapi/openapi.json

```diff
@@ -3138,16 +3128,20 @@
         "description": "The configuration for congestion control. More info about congestion [here](https://near.github.io/nearcore/architecture/how/receipt-congestion.html?highlight=congestion#receipt-congestion)",
         "properties": {
           "allowed_shard_outgoing_gas": {
-            "description": "How much gas the chosen allowed shard can send to a 100% congested shard.\n\nSee [`CongestionControlConfig`] for more details.",
-            "format": "uint64",
-            "minimum": 0,
-            "type": "integer"
+            "allOf": [
+              {
+                "$ref": "#/components/schemas/NearGas"
+              }
+            ],
+            "description": "How much gas the chosen allowed shard can send to a 100% congested shard.\n\nSee [`CongestionControlConfig`] for more details."
```

Comment:
Does it get flattened on the client generator side?

Reply:
No, why would it? You can see the example [here](https://github.com/PolyProgrammist/near-openapi-client/blob/1571520fc63970bd990d47a0b1d7058406e82351/near-openapi-types/src/lib.rs#L4448), NearGas is [pub use near_gas::NearGas](https://github.com/PolyProgrammist/near-openapi-client/blob/1571520fc63970bd990d47a0b1d7058406e82351/near-openapi-types/src/lib.rs#L5)